1208-projects/project_landing_page.feature

```diff
@@ -37,9 +66,10 @@ Feature: Project Landing Page
             | Name        |
             | Role        |
             | Affiliation |
+            | Start Date  |
```

Comment:
Skal start date vises for alle bidragsytere? mener å ha hørt at det bare er relevant for Project Manager tidligere

Reply:
Ja, det er kun project manager som har start date, alle andre participants antar å ha start og slutt dato tilsvarende som prosjektet har. Eg legger til ein kommentar på dette scenario:
# Only Project Manager has a start date

Reply:
Da høres det ut som at dette vil feile for "vanlige" Participants, siden det feltet ikke vil vises. Lurer på om vi skulle hatt eget scenario for Project Manager... blir dette riktig?
```
    @2632
    Scenario: User opens Participants for a Project
        Given User opens Landing Page for Project
        When they expand "Participants"
        Then they see a list of Participants and their:
            | Name        |
            | Role        |
            | Affiliation |

    Scenario: User sees Project Manager for a Project
        Given User opens Participants for a Project
        When they see a Project Manager
        Then they see fields:
            | Start Date  |
```

Reply:
@Eiriknil bør se på dette. kanskje holder det med ett scenario hvis vi lister opp rolle og felter i Examples?

Reply:
Eit eller to scenario kan Eirik avgjere. 
Funksjonelt så er Kjetils forslag dekkende.

Reply:
Tror jeg vil ha 2 scenarioer, er lettere å skrive en god test

Reply:
Done.